Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,83 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
+  local IFS=$'\n'
+  local basedir="${1/\~/$HOME}"
+  _values -C "packages from $basedir" ${$( find -L "$basedir" -mindepth 1 -maxdepth 1 \( -name .git -o -name .svn \) -prune -o -type d -print 2>/dev/null | sed "s#^${basedir}/##" | sort -u ):-""}
+}
+
+_stow(){
+  local context state line curcontext="$curcontext" ret=1
+  typeset -A opt_args
+  # Others local variables
+  local stow_dir arguments
+
+  arguments=(
+    '(- *)'{--help,-h}'[show help]'
+    '(- *)'{--version,-v}'[show version number]'
+    '(-d --dir)'{-d,--dir=-}'[set the stow dir (default is current dir)]:stow dir:_files -/'
```

Comment:
Is `{-d,--dir=-}` correct?  GNU programs generally use `{-d+,--dir=}` (added plus, removed minus).

Reply:
You're right, I'll fix that and do the same for `-t,--target=-`

Reply:
There are still a few instances of `--foo=-` in _stow and _chkstow.  Could you please confirm that they're correct, or change them if they aren't?

Reply:
Argh, good point.